app/src/main/kotlin/com/github/gotify/settings/SettingsActivity.kt

```diff
@@ -97,6 +99,17 @@ internal class SettingsActivity : AppCompatActivity(), OnSharedPreferenceChangeL
                     Utils.setExcludeFromRecent(requireContext(), value as Boolean)
                     return@OnPreferenceChangeListener true
                 }
+            findPreference<SwitchPreferenceCompat>(
+                getString(R.string.setting_key_intent_dialog_permission)
+            )?.let {
+                it.setOnPreferenceClickListener {
+                    openSystemAlertWindowPermissionPage()
+                }
+                it.setOnPreferenceChangeListener { _, _ ->
+                    openSystemAlertWindowPermissionPage()
+                }
```

Comment:
I think this triggers the twice when the item is clicked.

[Peek 2023-10-05 18-37.webm](https://github.com/gotify/android/assets/14895212/87573845-79b9-47e2-ab8c-952ddaa240e3)

Reply:
Apparently I'm dumb.
Fixed